pandas/core/base.py

```diff
@@ -990,11 +990,17 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
-        """ np.ndarray searchsorted compat """
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
doc strings ARE inherited if the subs don't have doc strings (but they do now) that's why u don't see it 

the sorter is passed onto np.searchsorter I think

ok so in that case - you can prob futz with it a bit and just pass theu sorter